core/extension/extension_api_dump.cpp

```diff
@@ -880,6 +880,15 @@ Dictionary GDExtensionAPIDump::generate_extension_api() {
 						d2["is_virtual"] = false;
 						d2["hash"] = method->get_hash();
 
+						Vector<uint32_t> compat_hashes = ClassDB::get_method_compatibility_hashes(class_name, method_name);
+						if (compat_hashes.size()) {
+							Array compatibility;
+							for (int i = 0; i < compat_hashes.size(); i++) {
+								compatibility.push_back(compat_hashes[i]);
+							}
+							d["hash_compatibility"] = compatibility;
```

Comment:
Thanks for this! Yeah it may be possible to also add just a hash, but IMO if we change the format too much I think that will have to be automated (as in, have a function that computes the old hash and add it as an option). For this specific case its not terrible, though.